src/main/java/teammates/ui/controller/InstructorCourseEditSaveAction.java

```diff
@@ -0,0 +1,45 @@
+package teammates.ui.controller;
+
+import teammates.common.datatransfer.CourseAttributes;
+import teammates.common.datatransfer.InstructorAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Assumption;
+import teammates.common.util.Const;
+import teammates.common.util.Sanitizer;
+import teammates.common.util.StatusMessage;
+import teammates.common.util.Const.StatusMessageColor;
+import teammates.logic.api.GateKeeper;
+
+public class InstructorCourseEditSaveAction extends Action {
+    @Override
+    protected ActionResult execute() throws EntityDoesNotExistException {
+        String courseId = getRequestParamValue(Const.ParamsNames.COURSE_ID);
+        Assumption.assertPostParamNotNull(Const.ParamsNames.COURSE_ID, courseId);
+        
+        String courseName = getRequestParamValue(Const.ParamsNames.COURSE_NAME);
+        Assumption.assertPostParamNotNull(Const.ParamsNames.COURSE_NAME, courseName);
+
+        InstructorAttributes instructor = logic.getInstructorForGoogleId(courseId, account.googleId);
+        new GateKeeper().verifyAccessible(instructor, logic.getCourse(courseId), Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_COURSE);
+        
+        CourseAttributes courseToEdit = new CourseAttributes();
+        courseToEdit.id = Sanitizer.sanitizeName(courseId);
```

Comment:
Hmm yea okay i think should be sanitized too coz both the id and name is not being sanitized when adding a new instructor... so I remove the sanitize for the courseId and courseName? @kanghj